pkgs/applications/version-management/gitstatus/default.nix

```diff
@@ -11,7 +18,7 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-b+9bwJ87VV6rbOPobkwMkDXGH34STjYPlt8wCRR5tEc=";
   };
 
-  buildInputs = [ (callPackage ./romkatv_libgit2.nix { }) ];
+  buildInputs = [(callPackage ./romkatv_libgit2.nix {})];
```

Comment:
We are using nixfmt-rfc-style for formatting which undo this again.

```
diff --git a/pkgs/applications/version-management/gitstatus/default.nix b/pkgs/applications/version-management/gitstatus/default.nix
index 931acbefb597..1d885bd5bb42 100644
--- a/pkgs/applications/version-management/gitstatus/default.nix
+++ b/pkgs/applications/version-management/gitstatus/default.nix
@@ -18,7 +18,7 @@
     sha256 = "sha256-b+9bwJ87VV6rbOPobkwMkDXGH34STjYPlt8wCRR5tEc=";
   };

-  buildInputs = [(callPackage ./romkatv_libgit2.nix {})];
+  buildInputs = [ (callPackage ./romkatv_libgit2.nix { }) ];

   postPatch = ''
     sed -i '1i GITSTATUS_AUTO_INSTALL=''${GITSTATUS_AUTO_INSTALL-0}' gitstatus.plugin.sh
@@ -64,7 +64,10 @@
   # that the script was sourced successfully and that the "gitstatus_query"
   # command ran successfully. This tests the binary itself and the zsh
   # integration.
-  nativeInstallCheckInputs = [git zsh];
+  nativeInstallCheckInputs = [
+    git
+    zsh
+  ];
   doInstallCheck = true;
   installCheckPhase = ''
     TEMP=$(mktemp -d)
@@ -121,7 +124,11 @@
     '';
     homepage = "https://github.com/romkatv/gitstatus";
     license = licenses.gpl3Only;
-    maintainers = with maintainers; [mmlb hexa SuperSandro2000];
+    maintainers = with maintainers; [
+      mmlb
+      hexa
+      SuperSandro2000
+    ];
     platforms = platforms.all;
     mainProgram = "gitstatusd";
   };
```

Reply:
Reformatted with nixfmt